sw/device/silicon_creator/lib/shutdown.c

```diff
@@ -215,16 +215,7 @@ uint32_t shutdown_redact(rom_error_t reason, shutdown_error_redact_t severity) {
  * This function must be inlined because it is called from `shutdown_finalize`.
  */
 static ALWAYS_INLINE shutdown_error_redact_t
-shutdown_redact_policy_inline(void) {
-  // Determine the error code redaction policy to apply according to the
-  // lifecycle state and OTP configuration.
-  //
-  // Note that we cannot use the lifecycle or OTP libraries since an error
-  // may trigger a call to `shutdown_finalize`.
-  uint32_t raw_state =
-      bitfield_field32_read(abs_mmio_read32(TOP_EARLGREY_LC_CTRL_BASE_ADDR +
-                                            LC_CTRL_LC_STATE_REG_OFFSET),
-                            LC_CTRL_LC_STATE_STATE_FIELD);
```

Comment:
Hmm, it's not clear to me why this read is moved out of this function.

Reply:
I moved it out to be able to reuse the value in the print statement. But I could also read the register twice, wdyt?

Reply:
Reading twice SG. We can also remove the lc_state print from here and move it somewhere else after log/uart/pinmux is updated.

Reply:
Actually, this will also depend on if we want to disable the message based on lifecycle state.